Assets/MixedRealityToolkit/InputModule/Scripts/InputEventData/BoundingBoxRigActivityEventData.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using MixedRealityToolkit.InputModule.InputSources;
+using UnityEngine.EventSystems;
+using UnityEngine;
+
+namespace MixedRealityToolkit.InputModule.EventData
+{
+    /// <summary>
+    /// Describes activity of a bounding box rig events.
+    /// </summary>
+    public class BoundingBoxRigActivityEventData : BaseInputEventData
+    {
+        /// <summary>
+        /// The bounding box rigged object.
+        /// </summary>
+        public GameObject BoundingBoxRiggedObject { get; private set; }
+
+        public BoundingBoxRigActivityEventData(EventSystem eventSystem) : base(eventSystem) { }
+
+        public void Initialize(IInputSource source, uint sourceId, object tag, GameObject boundingBoxRiggedObject)
```

Comment:
Tag should be an object array and last, as it's optional.

`public void Initialize(IInputSource source, uint sourceId, GameObject boundingBoxRiggedObject, object[] tags = null)